Automattic/wp-calypso
# Gutenberg blocks: use SVG primitives to add accessibility attributes

- Use [Gutenberg SVG primitives](https://github.com/WordPress/gutenberg/tree/ec1a4cbe126d860f0feb0dda8c0c8f8c1d38419b/packages/components/src/primitives/svg#readme) to add accessibility attributes, namely:
    ```
    role: 'img',
    'aria-hidden': 'true',
    focusable: 'false',
    ```

- Move `render-material-icon` utility from Jetpack preset to shared folder as it seems pretty generic utility and not specific to Jetpack.

#### Testing instructions

- Spin up Gutenberg in Jetpack: gutenpack-jn
- Insert affected blocks and confirm icons play nicely:
  - Form + forms innert blocks
  - Markdown
  - Map
  - Related posts
